### PR TITLE
Lossless 705 - Lossless Governance updates

### PR DESCRIPTION
**Pull request by Ignacio-Freire**

### CODE REVIEW

File: test/domain based tests/claimLosslessReward.js [hunk 1]
```diff
@@ -118,23 +117,23 @@ describe(scriptName, () => {
...
+      });
     });
-  });
 
-  describe('when lossless claim is called by not lossless admin two times', () => {
-    it('should revert', async () => {
-      await expect(
-        env.lssGovernance.connect(adr.reporter1).losslessClaim(1),
-      ).to.be.revertedWith('LSS: Must be admin');
+    describe('when lossless claim is called by not lossless admin two times', () => {
```
Comment (DomantasP) [inline test/domain based tests/claimLosslessReward.js:131]: In the test it seems that claim is called only one time?

File: test/domain based tests/claimRetrievedFunds.js [hunk 2]
```diff
@@ -87,5 +87,50 @@ describe(scriptName, () => {
...
+          env.lssGovernance.connect(adr.regularUser5).retrieveFunds(1),
+        ).to.not.be.reverted;
+
+        await expect(
+          env.lssGovernance.connect(adr.regularUser5).retrieveFunds(1),
+        ).to.be.revertedWith('LSS: Funds already claimed');
+      });
+    });
+
+    describe('when trying to retrieve by other than the reporter', () => {
```
Comment (DomantasP) [inline test/domain based tests/claimRetrievedFunds.js:110]: I think you meant: "... other than the proposed wallet"

File: test/domain based tests/claimRetrievedFunds.js [hunk 3]
```diff
@@ -87,5 +87,50 @@ describe(scriptName, () => {
...
+  describe('when dispute period is not over', () => {
+    it('should revert', async () => {
+      await env.lssGovernance.connect(adr.lssAdmin)
+        .proposeWallet(1, adr.regularUser5.address);
+
+      await expect(
+        env.lssGovernance.connect(adr.regularUser5).retrieveFunds(1),
+      ).to.be.revertedWith('LSS: Dispute period not closed');
+    });
   });
```
Comment (DomantasP) [inline test/domain based tests/claimRetrievedFunds.js:135]: Can we also add test cases where the proposed wallet is rejected and then it tries to claim? Or are they already added and I just don't see them in this commit?

File: test/domain based tests/disputeProposedWallet.js [hunk 4]
```diff
@@ -100,21 +100,60 @@ describe(scriptName, () => {
...
+        });
       });
 
-      it('should revert when trying to retrieve', async () => {
-        await expect(
-          env.lssGovernance.connect(adr.regularUser5).retrieveFunds(1),
-        ).to.be.revertedWith('LSS: Wallet rejected');
+      describe('when dispute period has closed', () => {
+        beforeEach(async () => {
+          await expect(
```
Comment (DomantasP) [inline test/domain based tests/disputeProposedWallet.js:132]: I think should be move to a separate describe block, looks odd to to test stuff in beforeEach block

File: test/domain based tests/claimLosslessReward.js (hunk 1 above)
Comment (Ignacio-Freire) [reply test/domain based tests/claimLosslessReward.js:131]: Addressed on 4bf0727

File: test/domain based tests/claimRetrievedFunds.js (hunk 2 above)
Comment (Ignacio-Freire) [reply test/domain based tests/claimRetrievedFunds.js:110]: Addressed on 27b3e7c

File: test/domain based tests/disputeProposedWallet.js (hunk 4 above)
Comment (Ignacio-Freire) [reply test/domain based tests/disputeProposedWallet.js:132]: Addressed on 3f3b943

**Pull request merged**